src/app/builder/builder.component.html

```diff
@@ -107,9 +111,16 @@
         </fieldset>
 
         <fieldset>
-          <img [src]="props.imageUrl || defaults.imageUrl" id="artwork-preview">
-          <label>Artwork URL</label>
-          <input type="text" [(ngModel)]="props.imageUrl" id="imageUrl" [placeholder]="defaults.imageUrl" name="imageUrl">
+          <img [src]="props.epImageUrl || defaults.epImageUrl" id="artwork-preview">
+          <label>Episode Artwork URL</label>
+          <p><small>This artwork will be featured prominently in the foreground of the player.</small></p>
+          <input type="text" [(ngModel)]="props.epImageUrl" id="epImageUrl" [placeholder]="defaults.epImageUrl" name="epImageUrl">
+        </fieldset>
+
+        <fieldset>
+          <label>Feed Artwork URL</label>
```

Comment:
Prior to this PR, the behavior was such that the `Artwork URL` would use the `itunes:image` from the episode if present, otherwise the `itunes:image` from the feed if present, otherwise nothing, right?

I guess I can see the value of allowing for the customization of both fg and bg, but I'm not sure I get why the bg would be the feed artwork, even if the selected episode has episode artwork.

Maybe I'm confused.